package.json

```
@@ -24,7 +24,10 @@
   "dependencies": {
     "base64url-universal": "^1.0.1",
     "bs58": "^4.0.1",
-    "node-forge": "^0.8.0",
+    "node-forge": "~0.9.0",
```

Can use `^` here since it's still under v1.
```suggestion
    "node-forge": "^0.9.0",
```

---

You really want to get me started about 0.x.x releases again!?  https://nodesource.com/blog/semver-tilde-and-caret/

Although for a 0.x.y release, ^ and ~ are functionally equivalent, I believe ~ more accurately depicts the behavior.